AndroidBound/src/main/java/solutions/alterego/androidbound/android/BoundFragmentDelegate.java

```diff
@@ -256,7 +256,7 @@ public void onDestroyView() {
                 && getViewBinder() != null
                 && boundActivityRef.getWindow() != null
                 && boundActivityRef.getWindow().getDecorView() != null) {
-            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView().getRootView());
+            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView());
```

Comment:
can't we call it from onDestroy of fragment delegate. We have a reference of the hosting activity, don't we ?